app/models/task.py

```diff
@@ -4,3 +4,8 @@
 
 class Task(db.Model):
     task_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'), nullable = True)
```

Comment:
Our tests don't confirm this, but should tasks be able to store `NULL` titles and descriptions? 

All columns are nullable by default, so we can remove the `nullable = True` attribute from the `completed_at` column and it will work the same way.